app/assets/javascripts/setup.js

```diff
@@ -0,0 +1,5 @@
+$('.btn-toggle').click(function() {
```

Comment:
I would rename it to `.btn-group-toggle` or just add `.btn-group` to make it clear that `.btn-toggle` is in the same level of a button group component.

If `.btn-group-toggle` is the choice, remember to update the view template.